overlord/overlord.rb

```
@@ -1,14 +1,56 @@
-# run `ruby overlord.rb` to run a webserver for this app
-
 require 'sinatra'
+require_relative 'bomb'
 
 enable :sessions
 
 get '/' do
-  "Time to build an app around here. Start time: " + start_time
+  DEFAULT_ACTIVATION_CODE = Bomb::DEFAULT_ACTIVATION_CODE
+  DEFAULT_DEACTIVATION_CODE = Bomb::DEFAULT_DEACTIVATION_CODE
+  erb :boot
+end
+
+post '/' do
+  activation_code = params['activation_code']
+  deactivation_code = params['deactivation_code']
+  if Bomb.valid_code?(activation_code) && Bomb.valid_code?(deactivation_code)
```

There is no need for this conditional statement. When creating the bomb in the statement below it should raise the error when it's initialized.